### ISSUE TITLE
[BUG] Waydroid doesn't work on the dedicated AMD GPU

### ISSUE THREAD

**Issue by Rigerhit047** [waydroid/waydroid#1278]
### Describe the bug

So, I have a dual GPU setup (integrated + discrete) on a desktop pc and I was able to run waydroid fine before, but I noticed that the integrated gpu was the main one used by everything, so after some tinkering is it now the main one, but after doing that I noticed that waydroid doesn't react to anything at all, I tried removing all the files, reinstalled multiple times, tried to change the prop settings, and nothing, the OTA window doesn't even show up and I can't even get log since "container is stopped" although the service IS active, tried reading some older issues and tried some stuff like the prop and the script to change gpu to no avail, it seems that AMD GPUs should be support based on those issues, so I'm very confused and I don't really know what to do...

Do note that when switching to the igpu as main one again makes it work perfectly again, but it ruins my experience in other stuff so I would like to have my dGPU as main. These logs are from a fresh reinstall.

### Waydroid version

1.4.2

### Device

Linux Desktop

### Operating System

Fedora Linux 39 (Workstation Edition)

### Kernel version

6.7.4-200.fc39.x86_64

### Desktop Environment

GNOME 45.3

### GPU

AMD Radeon™ RX 570 Series/Intel® HD Graphics 4600 (HSW GT2) from Intel® Core™ i5-4690K × 4

### Logs

[waydroid.log](https://github.com/waydroid/waydroid/files/14237031/waydroid.log)

[dmesg.txt](https://github.com/waydroid/waydroid/files/14237027/dmesg.txt)

[waydroid.zip](https://github.com/waydroid/waydroid/files/14237055/waydroid.zip)

**Comment by adocampo** [comment]
How do you change to iGPU, I cannot see the videos as per https://github.com/waydroid/waydroid/discussions/855#discussioncomment-8610325 and I would like to see if changing from NVIDIA to iGPU does anything. I found this script pointed in the documentation, but it only shows my two NVIDIA cards, not the GPU embedded on my Intel's CPU.

**Comment by Rigerhit047** [comment]
@adocampo I just changed the default gpu on bios to the dedicated one, the option can be named like that or instead it would be "Primary Monitor" although for what you are saying it seems that your iGPU is not really enabled (or it doesn't have one), so you would need to do that in the BIOS as well. If that doesn't work I think there was some scripts to change it at boot, but make sure to actually enable the iGPU first. Sorry for not answering you.

Also I'm closing this issue in favor of #1322 since it is better explained and has more people relating to the issue. I also made a comment with the workaround I did to get it working.
